packages/configs/index.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+	root: true,
+	parser: '@typescript-eslint/parser',
+	env: {
+		node: true,
+	},
+	plugins: ['@typescript-eslint'],
+	extends: [
+		'prettier',
+		'plugin:prettier/recommended',
+		'eslint:recommended',
+		'plugin:@typescript-eslint/recommended',
+		'plugin:@typescript-eslint/recommended-requiring-type-checking',
```

Comment:
Will this break if it tries to lint a project without typescript? Do we need to remove these for this first publish?

Reply:
Seems to still work in projects without typescript. I used `npm link` to bring this package into a new create react app project outside of the monorepo and experienced no issues.

Reply:
Great, thank you for confirming